pages/slider/permutations.page.tsx

```diff
@@ -17,6 +17,26 @@ const permutations = createPermutations<SliderProps>([
     invalid: [false, true],
     step: [0.5, 1, 5],
   },
+  {
+    value: [-10, 0, 10],
+    min: [-10],
+    max: [10],
+    hideFillLine: [false, true],
+    disabled: [false],
+    tickMarks: [false, true],
+    warning: [true],
+    step: [0.5, 1, 5],
+  },
+  {
+    value: [0],
```

Comment:
It seems like this is doing the same as the above. Can you combine them and add `disabled: [false, true]` to the top one?

Reply:
The one below tests the warning slider with disabled setting, combining them exceeds the page limitation that's why I separated this part.